src/components/DimensionsPanel/__tests__/DimensionsPanel.spec.js

```diff
@@ -1,43 +1,56 @@
-import { shallow } from 'enzyme'
+import { render, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
 import React from 'react'
 import DimensionsPanel from '../DimensionsPanel.js'
-import DimensionList from '../List/DimensionList.js'
-
-describe('The Dimensions component ', () => {
-    let shallowDimensions
-    let props
-    const dimensionsComponent = () => {
-        if (!shallowDimensions) {
-            shallowDimensions = shallow(<DimensionsPanel {...props} />)
-        }
-        return shallowDimensions
-    }
-
-    beforeEach(() => {
-        shallowDimensions = undefined
-        props = {
-            dimensions: [],
-        }
-    })
-
-    it('renders a div', () => {
-        expect(dimensionsComponent().find('div').length).toEqual(1)
-    })
-
-    it('renders a div containing everything else', () => {
-        const wrappingDiv = dimensionsComponent().find('div').first()
-
-        expect(wrappingDiv.children()).toEqual(dimensionsComponent().children())
-    })
-
-    it('renders a DimensionList with the correct prop', () => {
-        const dimensionsComp = dimensionsComponent()
-        dimensionsComp.setState({ filterText: 'filteredText' })
-
-        const filteredList = dimensionsComp.find(DimensionList).first()
-
-        expect(filteredList.props().filterText).toEqual(
-            dimensionsComp.state().filterText
-        )
-    })
+
+const props = {
+    dimensions: [
+        { id: 'id1', name: 'Dimension 1' },
+        { id: 'id2', name: 'Dimension 2' },
+    ],
+}
+
+test('DimensionsPanel matches the snapshot', () => {
+    const { container } = render(<DimensionsPanel {...props} />)
+
+    expect(container).toMatchSnapshot()
```

Comment:
I think maybe just remove this test since the snapshots has all the jsx classes from the ui library (e.g., "jsx-1141087211"). This will cause us to sometimes need to regenerate snapshot files when we upgrade ui. Also, I think the next test more or less covers things (you could add the expectation of "Filter dimensions" on line 42 to the next test)

Reply:
Removed the snapshot test.